src/components/BSelect/BSelect.vue

```diff
@@ -0,0 +1,284 @@
+<template>
+  <div>
+    <div>
+      <label>{{ selectLabel }} <span v-if=isRequired aria-label='Required'>*</span></label>
+    </div>
+    <div 
+      :aria-expanded="opened"
+      :aria-label="selectLabel"
+      :aria-owns="id"
+      :aria-required="isRequired"
+      :id="'dropdown' + id"
+      @keyup.32.prevent.stop="toggleList"
+      role="combobox"
+      tabindex="0"
+      v-model="selectLabel"
+    >
+      <ul
+        :class="ulClass"
+        :id="id"
+        role="listbox"
+      >
+        <li 
+          :class="dropdownClass" 
+          @click="toggleList"
+          >{{ selectedOption ? selectedOption.text : placeholder }}
+          <span class="dropdown-arrow"><i class="fa fa-caret-down" aria-hidden="true"></i></span>
+        </li>
+        <span v-show="selectErrors.has('selectedOption')" class="error" data-vv-as="selectLabel">{{ selectErrors.first('selectedOption') }}</span>
+        <li
+          v-if=opened
+          v-for="option in options"
+          :aria-selected="selectedOption && selectedOption.id === option.id"
+          :id="'option' + option.id"
+          :key="'selectOption' + option.id"
+          :value=option.id
+          @click="selectOption(option)"
+          @keyup.38.prevent="upHandler"
+          @keyup.40.prevent="downHandler"
+          @keyup.13.prevent="selectOption(option)"
+          @keyup.32.prevent.stop="selectOption(option)"
+          class="li-opened options"
+          role="option"
+          tabindex='-1'
+        > {{ option.text }}
+        </li>
+      </ul>
+    </div>
+  </div>
+</template>
+
+<script>
+
+import { Validator } from 'vee-validate'
+
+const options = []
+
+export default {
+  name: 'b-select',
+  validator: null,
+  props: {
+
+    id: {
+      type: String,
+      required: true
+    },
+    selectOptions: {
+      type: Array
+    },
+    isRequired: {
```

Comment:
Why not just `required` so it looks more like the actual element. You could then also make it look like `<b-select required></b-select>` instead of `<b-select :is-required="true"></b-select>` or `<b-select is-required></b-select>`

Reply:
i thought that would be more confusing.  This isn't an select element that is required or not, its a un with li elements.

Reply:
I think it should probably be consistent with all the others because it still technically is a from-type component.

Reply:
Right, and to the user it _looks_ like a select element so we should provide those options consistently across elements.